### PR TITLE
docs(sv-61): initial guide and readme files

### PR DESCRIPTION
**Pull request by lukasborawski**
### 🔗 Linked issue

[https://vsf.atlassian.net/browse/SV-61](SV-61)

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This PR provides Boilerplate README and GUIDE files with related to the Nuxt/Vue contents.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+- TypeScript types and tests are located close to the function file.
+
+Expected file/folder structure:
+
+```shell
+<module-directory>/
+ └── Function/
+     ├─ index.ts
+     ├─ types.ts
+     ├─ Function.tsx
```
Comment (dawid-ziobro) [inline GUIDE.md:117]: `tsx` used. If this is about components maybe rename to `Component.vue`?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,85 @@
...
 
-- `Node.js` 20+
-- `Yarn` 3
+- `Node.js` 18+
+- `Yarn`
 
 ### Local dev setup
 
-1. Clone the repository.
+1. Clone the repo
```
Comment (FRSgit) [inline README.md:28]: repository was more descriptive 😄

File: GUIDE.md (hunk 1 above)
Comment (dawid-ziobro) [reply GUIDE.md:117]: Ok i can see components down below

Comment (lukasborawski) [reply GUIDE.md:117]: That's wrong. Will fix that.

Comment (FRSgit) [reply GUIDE.md:117]: Yup, there are few places where `tsx` is used, please just replace them :shrug:

File: README.md (hunk 2 above)
Comment (lukasborawski) [reply README.md]: Yes. Will change that.

Comment (lukasborawski) [reply README.md]: Already done, actually. :P

File: README.md [hunk 3]
```diff
@@ -1,24 +1,85 @@
...
+
+Refer to [Guide](GUIDE.md)
+
+## License
+
+Distributed under the MIT License. See [LICENSE](LICENSE.md) for more information.
+
+## Contributing
+
+Please see our [contributing.md](contributing.md) for more information.
```
Comment (dawid-ziobro) [inline README.md:74]: Do we have this file anywhere?

Comment (lukasborawski) [reply README.md:74]: It should be added automatically by the global template.

Comment (dawid-ziobro) [reply README.md:74]: Ah ok, i thought about that but i wasn't sure. Thanks!

**Review by dawid-ziobro — APPROVED** [review]

**Pull request merged**